lib/redis/store/namespace.rb

```diff
@@ -39,6 +39,17 @@ def keys(pattern = "*")
         namespace(pattern) { |p| super(p).map { |key| strip_namespace(key) } }
       end
 
+      def scan(cursor, options = {})
+        if options[:match]
+          namespace(options[:match]) do |p|
```

Comment:
what does this `p` stand for? can we make the block arg a little more descriptive?